tensorflow/python/data/util/structure_test.py

```diff
@@ -73,14 +72,14 @@ class StructureTest(test_base.DatasetTestBase, parameterized.TestCase,
           "b": constant_op.constant([1, 2, 3])
       }, dict, [dtypes.float32, dtypes.int32], [[], [3]]),
       ("Nested_2", lambda: {
-          "a":
-              constant_op.constant(37.0),
+          "a": constant_op.constant(37.0),
```

Comment:
You should avoid reformatting code that you are not changing.

Reply:
addressed in the commit.